src/rpdk/core/jsonutils/utils.py

```diff
@@ -103,6 +103,95 @@ def traverse(document, path_parts):
     return document, tuple(path), parent
 
 
+# pylint: disable=R0913,W0102,C0301
```

Comment:
please try to leave some comments on why these are disabled. why so many arguments? can this be refactored to avoid all the args